README.md

```diff
@@ -176,18 +184,24 @@ The following table lists the supported annotations for Kubernetes `Deployments`
 
 | Annotation | Description | Default |
 | ---------- | ----------- | ------- |
-| `osiris.deislabs.io/enabled` | Enable Osiris for this deployment. Allowed values: `y`, `yes`, `true`, `on`, `1`. | _no value_ (= disabled) |
+| `osiris.deislabs.io/enabled` | Enable Osiris for this deployment. Allowed values: `y`, `yes`, `true`, `on`, `1`. | _no value_ (= disabled). Enabling this ensures the zeroscaler component will scrape and analyze metrics from the deployment's pods. |
```

Comment:
shouldn't this new sentence be written in the `description` column, instead of the `default` one?